plugins/git_stats.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import re
+from shutil import rmtree
+
+from errbot import re_botcmd
+
+from plugins.labhub import LabHub
+
+
+class GitStats(LabHub):
+    """GitHub and GitLab statistics"""  # Ignore QuotesBear
+
+    PR_LABELS = ('process/approved',
+                 'process/pending review',
+                 'process/pending-review'
+                 )
+
+    def __init__(self, bot, name=None):
+        super().__init__(bot, name)
+
+    @re_botcmd(pattern=r'mergable\s+([^/]+)',  # Ignore PyCodeStyleBear
+               re_cmd_name_help='pr list <repo>',
+               flags=re.IGNORECASE)
+    def pr_list(self, msg, match):
+        """List PRs ready to be merged."""  # Ignore QuotesBear
+        repo_name = match.groups(1)[0]
+
+        try:
+            mrs = self.REPOS[repo_name].merge_requests()
+        except KeyError:
+            return "Repository doesn't exist."
+
+        checks = []
+
+        def check_mr(function):
+            checks.append(function)
+            return function
+
+        @check_mr
+        def check_labels(mr):
+            labels = set(mr.labels)
+            if labels.intersection(self.PR_LABELS):
+                return True
+            return False
+
+        @check_mr
+        def check_state(mr):
+            if mr.state == 'open':
+                return True
+            return False
+
+        @check_mr
+        def check_rebased(mr):
+            repo, tempdir = mr.repository.get_clone()
+            head_sha = repo.head.commit.hexsha
+            rmtree(tempdir)
+            if mr.base.sha == head_sha:
+                return True
+            return False
+
+        def merge_ready(mr):
+            for chk in checks:
+                if not chk(mr):
+                    return False
+            return True
+
+        rdy = []  # List of mergable PRs
```

Comment:
Please use real words as variable names, with 'snake case'.  See PEP 8 coding standard.

Reply:
Yup, got it fixed